apps/patient/src/utils/models.ts

```diff
@@ -2,18 +2,29 @@ import {
   GetPharmaciesByLocationQuery,
   GetOrderQuery,
   Address as GQLAddress,
-  FulfillmentType
+  FulfillmentType,
+  Maybe
 } from '../__generated__/graphql';
 
 type NotMaybe<T> = Exclude<T, null | undefined>;
 export type Order = NotMaybe<GetOrderQuery['order']>;
 export type Fill = Order['fills'][number];
+export interface DiscountCard {
+  id: string;
+  price: string;
+  bin: string;
+  pcn: string;
+  group: string;
+  memberId: string;
+}
```

Comment:
I would pull this off `Order` rather than hardcoding like this.

Reply:
oh think this is leftover from pre-codegen